workflows/DATA_TO_MODEL.nf

```diff
@@ -29,18 +33,15 @@ include { ANNOTATE_SUBMISSION as ANNOTATE_SUBMISSION_AFTER_SCORE } from '../modu
 include { SEND_EMAIL } from '../modules/send_email.nf'
 
 workflow DATA_TO_MODEL {
+    submission_ch = CREATE_SUBMISSION_CHANNEL()
     SYNAPSE_STAGE(params.testing_data, "testing_data")
-    GET_SUBMISSIONS(params.view_id)
-    image_ch = GET_SUBMISSIONS.output 
-        .splitCsv(header:true) 
-        .map { row -> tuple(row.submission_id, row.image_id) }
-    UPDATE_SUBMISSION_STATUS_BEFORE_RUN(image_ch.map { tuple(it[0], "EVALUATION_IN_PROGRESS") })
-    DOWNLOAD_SUBMISSION(image_ch.map {it[0]})
+    UPDATE_SUBMISSION_STATUS_BEFORE_EVALUATION(submission_ch, "EVALUATION_IN_PROGRESS")
```

Comment:
One thing I realized is that this will be redundant during automated runs (since Airflow will do this). I kinda think we should leave it in for non-automated runs though.